grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -673,12 +674,13 @@ class NotifyService implements SendgridSupport, Status {
         sendMail(mail, getApiKey(), getSendGridUrl())
     }
 
-    Map<Boolean, String> sendEditsDisapprovedNotification(Issue issue) {
+    Map<Boolean, String> sendEditsDisapprovedNotification(Issue issue, String editCreatorName) {
         String type = issue.type?.equals(IssueType.CONSENT_GROUP.getName()) ? "Consent Group" : "Project"
         User user = userService.findUser(issue.reporter)
         NotifyArguments arguments =
                 new NotifyArguments(
                         toAddresses: Collections.singletonList(user.emailAddress),
+                        ccAddresses: editCreatorName != null ? Collections.singletonList(userService.findUser(editCreatorName).emailAddress) : [""],
```

Comment:
Note that this only works because we do a lot of validation on recipients here:
```
    private List<String> filterRecipients(Collection<String> recipients) {
        if (recipients?.contains(ORSP_ADDRESS)) {
            recipients -= ORSP_ADDRESS
            recipients += getOrspSpecialRecipients()
        }
        if (recipients != null) {
            recipients.toSet()
            recipients -= null
            recipients.toList()
        } else {
            Collections.emptyList()
        }
    }
```
TOL ... (thinking out loud) ... would it be better to just pass an empty array in the null case instead of an array with an empty string value?

Reply:
The more I ponder this, the more I think passing an empty array makes more semantic sense. Your code should never try to send an email to `""` ... and that's what this code is saying.

Reply:
Makes sense. As you suggested, this condition returns an empty array for null case. I did a local test round and it works good.